build.gradle

```diff
@@ -1,93 +1,286 @@
-ext.dockerImageName = "cli"
-ext.dockerContainerName = "cli"
-ext.dockerBuildArgs = getDockerBuildArgs()
-apply from: 'gradle/docker.gradle'
+// plugins {
```

Comment:
Remove commented lines here.

Reply:
Actually, would like to hold them as they'll be restored when gogradle 0.8.1 comes out.  Added a note about that.